packages/integration/lib/src/views/feedroom/feedroom_list_screen.dart

```diff
@@ -0,0 +1,197 @@
+            }
+            return Scaffold(
+              backgroundColor: LMFeedCore.theme.backgroundColor,
+              body: LMFeedLoader(),
+            );
+          }),
+        ),
+      ),
+    );
+  }
+}
+
+Widget getFeedRoomListEmptyView() {
+  return const Center(
+    child: Text("No feedrooms found"),
+  );
+}
+
+Widget getFeedRoomListErrorView(String message) {
+  return Center(
+    child: Text(message),
+  );
+}
+
+Widget getFeedRoomListLoadingView() {
+  return const LMFeedLoader();
+}
+
+class FeedRoomListView extends StatelessWidget {
+  final LMFeedRoomBloc feedRoomBloc;
+  final PagingController<int, LMFeedRoomViewData> pagingControllerFeedRoomList;
+  const FeedRoomListView({
+    super.key,
+    required this.pagingControllerFeedRoomList,
+    required this.feedRoomBloc,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const SizedBox(height: 18),
+        Expanded(
+          child: PagedListView<int, LMFeedRoomViewData>(
+            pagingController: pagingControllerFeedRoomList,
+            builderDelegate: PagedChildBuilderDelegate<LMFeedRoomViewData>(
+                noMoreItemsIndicatorBuilder: (context) =>
+                    const SizedBox(height: 20),
+                noItemsFoundIndicatorBuilder: (context) => Scaffold(
+                      backgroundColor: LMFeedCore.theme.backgroundColor,
+                      body: LMFeedLoader(),
+                    ),
+                itemBuilder: (context, item, index) {
+                  return LMFeedTile(
+                    onTap: () {
+                      Navigator.of(context).push(
+                        MaterialPageRoute(
+                          builder: (context) => LMFeedRoomScreen(
+                            feedroomId: item.id,
+                            // feedRoomTitle: item.title,
+                          ),
+                        ),
+                      );
+                    },
+                    leading: LMFeedProfilePicture(
+                      fallbackText: item.title,
+                      imageUrl: item.chatroomImageUrl,
+                      style: LMFeedProfilePictureStyle(
+                        size: 64,
+                        boxShape: BoxShape.circle,
+                      ),
+                    ),
+                    title: LMFeedText(
+                      text: item.header,
+                      style: LMFeedTextStyle(
+                        textStyle: TextStyle(
+                          fontSize: 18,
+                          fontWeight: FontWeight.w500,
+                        ),
+                      ),
+                    ),
+                    subtitle: LMFeedText(
+                      text: "${item.participantsCount} participants",
+                      style: LMFeedTextStyle(
+                        textStyle: const TextStyle(
+                          fontStyle: FontStyle.italic,
```

Comment:
we should add colors in text as well `theme.onContainer` so that's it's customisable through theme.